### PR TITLE
Add images and attachments for Extending the Identity Server doc section

### PR DESCRIPTION
**Pull request by YvonneW**
The following images were renamed and added in relation to the **Extending the Identity Server** doc section:

**User Store Listeners** 
- `/assets/img/103330086/103330088.png` ==> `/assets/img/using-wso2-identity-server/user-store-operations.png`

**Carbon Remote User Store Manager**
- `../../assets/img/103330074/103330077.png` ==> `../../assets/img/using-wso2-identity-server/add-new-user-store.png`

**Extensible SCIM User Schemas With WSO2 Identity Server** 
- `../../assets/img/29923055/29906299.png` ==> `../../assets/img/using-wso2-identity-server/sample-user-object.png`
- `../../assets/img/29923055/29906301.png` ==> `../../assets/img/using-wso2-identity-server/sample-extended-user-object.png`
- `../../assets/img/29923055/29906302.png` ==> `../../assets/img/using-wso2-identity-server/new-user-attributes.png`
- `../../assets/img/29923055/29906303.png` ==> `../../assets/img/using-wso2-identity-server/created-scim-user-object.png`

**Writing a Custom Local Authenticator**
- `../../assets/img/103330069/103330070.png` ==> `../../assets/img/using-wso2-identity-server/oauth-openid-connect-configuration-option.png`

**Writing a Custom Event Handler**
- `../../assets/img/103330095/103330097.png` ==> `../../assets/img/using-wso2-identity-server/handler-hierarchy.png`
- `../../assets/img/103330095/103330098.png` ==> `../../assets/img/using-wso2-identity-server/operation-category-selection.png`
- `../../assets/img/103330095/103330096.png` ==> `../../assets/img/using-wso2-identity-server/operation-category-name.png`

The following images were added in relation to the **Extending the Identity Server** doc section:
- `../../assets/img/callbacl-url.jpeg`
- `../../assets/img/using-wso2-identity-server/local-authentication-check-box.jpeg`
- `../../assets/img/using-wso2-identity-server/playground-app.jpeg` 

Renamed the following attachments and moved them to appropriate folders:

**Admin Services for One Way Operations** 
- `../../assets/img/112364676/112364673.wsdl` ==> `../../assets/attachments/user-identity-management-admin-service.wsdl`
- `../../assets/img/112364676/112364672.wsdl` ==> `../../assets/attachments/remote-user-store-manager-service.wsdl`
- `../../assets/img/112364676/112364674.wsdl` ==> `../../assets/attachments/user-information-recovery-service.wsdl`
- `../../assets/img/112364676/112364675.wsdl` ==> `../../assets/attachments/authentication-admin-service.wsdl`
- `../../assets/img/112364676/112364671.wsdl` ==> `../../assets/attachments/challenge-question-management-admin-service.wsdl`

**Customizing the SSO Redirection Page** 
- `../../assets/img/103330131/103330132.html` ==> `../../assets/attachments/sample-sso-redirect-page-template.html`

### CODE REVIEW

File: en/docs/assets/attachments/sample-sso-redirect-page-template.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+			<p>Content Area for Service Provider Two</p>
+		</div>
+
+
+        	<form method='post' action='$acUrl'>
+       			<p>
+        			<input type='hidden' name='SAMLResponse' value='$response'>
+        			<input type='hidden' name='RelayState' value='$relayState'>
+                    <!--$additionalParams-->
```
Comment (jeradrutnam) [inline en/docs/assets/attachments/sample-sso-redirect-page-template.html]: Remove the commented line

File: en/docs/assets/attachments/sample-sso-redirect-page-template.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<!-- 
+Variables $acUrl, $response, $relayState and $additionalParams will be replaced by the corrosponding values  
+-->
+<html>
```
Comment (jeradrutnam) [inline en/docs/assets/attachments/sample-sso-redirect-page-template.html]: Fix indentations in the file. Use 4 spaces.

File: en/docs/assets/attachments/sample-sso-redirect-page-template.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        			<input type='hidden' name='RelayState' value='$relayState'>
+                    <!--$additionalParams-->
+        			<button type='submit'>POST</button>
+       			</p>
+       		</form>
+
+                 <script type='text/javascript'>
+        		var url = '$acUrl';
+			
+			if(url === "https://localhost:9453/acs")
```
Comment (jeradrutnam) [inline en/docs/assets/attachments/sample-sso-redirect-page-template.html]: Don't use hard coded urls. Instead use a placeholder and guide users on how to replace it with their value

File: en/docs/assets/attachments/sample-sso-redirect-page-template.html (hunk 2 above)
Comment (YvonneW) [reply en/docs/assets/attachments/sample-sso-redirect-page-template.html]: Fixed

File: en/docs/assets/attachments/sample-sso-redirect-page-template.html (hunk 3 above)
Comment (jeradrutnam) [reply en/docs/assets/attachments/sample-sso-redirect-page-template.html]: Will fix in the samples revisit iteration

**Pull request merged**